src/constants.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <vector>
+
+#define Consts Constants::Instance()
+class Constants
+{
+public:
+	std::vector<std::string> displays = { "Left", "Right", "Dispartiy" };
+
+	static Constants* Instance()

```

Comment:
C++11 includes smart pointers. Use `std::shared_pointer<Constants>` from `#include <memory>`. Everything else works exactly the same.